### PR TITLE
added a status tab and removed the context from urls.py

### PR DESCRIPTION
**Pull request by samcaspus**
## Changes
Resolve issue https://github.com/PostHog/posthog/issues/1808
upgraded all the functions to _status url 
Modified the test for the same

## Checklist

- [x] All querysets/queries filter by Team (if this PR affects any querysets/queries)
- [x] Backend tests (if this PR affects the backend)
- [ ] Cypress E2E tests (if this PR affects the front and/or backend)

### CODE REVIEW

**Comment by samcaspus** [comment]
@Twixes would you suggest me to remove the health, stat and preflight from views as well ? or letting the views still exist is okay ? and also should i modify the test ?

**Comment by samcaspus** [comment]
Implemented @Twixes  mentioned in https://github.com/PostHog/posthog/issues/1808#issuecomment-703238190 please let me know if any changes required CC @timgl

**Comment by samcaspus** [comment]
implemented the changes as suggested by @Twixes  please verify CC @timgl

File: posthog/urls.py [hunk 1]
```diff
@@ -204,9 +204,10 @@ def opt_slash_path(route: str, view: Callable, name: Optional[str] = None) -> st
 
 urlpatterns = [
     # internals
-    opt_slash_path("_health", health),
-    opt_slash_path("_stats", stats),
+    # opt_slash_path("_health", health),
```
Comment (timgl) [inline posthog/urls.py:207]: In general I am in favour of just removing code instead of commenting it!

File: posthog/urls.py [hunk 2]
```diff
@@ -204,9 +204,10 @@ def opt_slash_path(route: str, view: Callable, name: Optional[str] = None) -> st
...
+    # opt_slash_path("_stats", stats),
```
Comment (timgl) [inline posthog/urls.py:208]: This page is actually used in WorkerStats.js. However, I actually think we can get rid of that entirely as it doesn't show useful information and tends to be confusing to people. Do you want to take that on? Should be easy

**Review by timgl — COMMENTED** [review]
Some comments :)

File: posthog/urls.py (hunk 2 above)
Comment (samcaspus) [reply posthog/urls.py]: I have made the changes as you wanted I hope thats what you were looking for incase not let me know ill update the pr 
thanks @timgl

**Comment by samcaspus** [comment]
Not sure why the cypress test is failing could you please suggest me any changes that might be required?

**Pull request closed without merging**